spec/requests/api/v0/kafka_data_spec.rb

```diff
@@ -92,7 +95,21 @@
     )
   end
 
-  it 'includes the event type' do
-    expect(instance).to include(type: 'org.openstax.ec.started_session')
+  it 'compacts  the device_id' do
+    expect(instance).to include(
+      device_uuid: CompactUuid.pack(device_id)
+    )
+  end
+
+  it 'includes the event schema name' do
+    expect(instance.schema_name).to include('org.openstax.ec.started_session')
+  end
+
+  it 'returns nil for unknown key' do
+    expect(instance[:foo]).to be_nil
```

Comment:
Previously this returned the entire dump of the controller :-)

Reply:
Ah, the stubbing above is why I couldn't repro Tom's error in a spec, right?